org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -421,6 +421,24 @@ SsoWorkflowNextflowDevDeveloper:
     instanceArn: !Ref instanceArn
     principalId: !Ref workflowNextflowDevDeveloperGroup
     permissionSetArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-developer-permission-set-arn' ]
+    inlinePolicy: >-
+      {
+          "Version": "2012-10-17",
+          "Statement": [
+              {
+                  "Sid": "ResourceConstrainedIAM",
+                  "Effect": "Allow",
+                  "Action": "iam:*",
+                  "Resource": [
+                      "arn:aws:iam::035458030717:user/dev-*",
+                      "arn:aws:iam::035458030717:role/dev-*",
+                      "arn:aws:iam::035458030717:group/dev-*",
+                      "arn:aws:iam::035458030717:instance-profile/dev-*",
+                      "arn:aws:iam::035458030717:policy/dev-*"
```

Comment:
I'm not sure how this is going to work.  All SSO user ARNs look something like this.. `arn:aws:sts::035458030717:assumed-role/AWSReservedSSO_Developer_82620564a2842685/joe.smith@sagebase.org`.  This doesn't match SSO users so i'm not sure what these are referring to.  Are these place holders for resources that are yet to be created?  like service user/roles or something?

Reply:
This allows the developer user to create users, roles, groups, instance-profiles, and policies whose names begin with "dev-". The point of this is to allow the developer role to be able to develop but not harm -- for example, looking to avoid deletion of the AWS Reserved SSO roles.

Reply:
hmm, still don't understand but no matter.  we can merge and try.

Reply:
Cool -- I created this policy and tried it out independently before creating the PR. It worked for me. Let me know if you want me to explain further.